app/models/TaxSummaryLiability.scala

```diff
@@ -60,6 +60,7 @@ object TaxSummaryLiability extends Logging {
                 acc
               }
             }
+          case (acc, ("tliLastUpdated", _)) => acc
```

Comment:
This can go after line 47.

Then `case (acc, (key, value)) if key != "tliLastUpdated"` can be just `case (acc, (key, value))`